### PR TITLE
[sailfish-setup] Default user groups file. Fixes JB#48709

### PR DESCRIPTION
**Pull request by mkenttala**
Does that %pre to %post change affect image building?

### CODE REVIEW

File: rpm/sailfish-setup.spec [hunk 1]
```diff
@@ -33,33 +33,22 @@ if ! getent passwd sailfish-actdead >/dev/null ; then
...
-usermod -a -G sailfish-alarms nemo || :
 
 groupadd -rf sailfish-datetime || :
-usermod -a -G sailfish-datetime nemo || :
 
 groupadd -rf timed || :
-usermod -a -G timed nemo || :
+
+%install
+install -D group.ids $RPM_BUILD_ROOT%{_libdir}/setup/group.ids
```
Comment (Tomin1) [inline rpm/sailfish-setup.spec]: I think datadir would be better than libdir and sailfish-setup better than setup. Maybe like this:
```suggestion
install -D group.ids $RPM_BUILD_ROOT%{_datadir}/sailfish-setup/group.ids
```

File: rpm/sailfish-setup.spec [hunk 2]
```diff
@@ -33,33 +33,22 @@ if ! getent passwd sailfish-actdead >/dev/null ; then
...
 
 %files
 %defattr(-,root,root,-)
+%{_libdir}/setup/group.ids
```
Comment (Tomin1) [inline rpm/sailfish-setup.spec]: This should own the directory.

File: README.md [hunk 3]
```diff
@@ -36,3 +36,7 @@ Groups that this package defines are listed below.
   - For processes that need to be able to modify clock settings.
 - timed
   - For timed process, required for changing system default timezone
+
+Default system (SYSTEM_GROUPS) and additional user (USER_GROUPS) groups are
+stored in the file /usr/lib/setup/group.ids, system user will be created during
+boot if not already existing.
```
Comment (Tomin1) [inline README.md]: English sounds a bit poor here. Maybe it could be something like this:
```suggestion
boot if it does not already exist.
```

**Review by Tomin1 — CHANGES_REQUESTED** [review]

**Review by Tomin1 — APPROVED** [review]

File: rpm/sailfish-setup.spec [hunk 4]
```diff
@@ -33,33 +33,27 @@ if ! getent passwd sailfish-actdead >/dev/null ; then
...
-    # users (setup)
-    useradd -g nemo -G "video,input,audio,users" -u 100000 -m nemo || :
-fi
-
 groupadd -rf oneshot || :
-usermod -a -G oneshot nemo || :
 
 groupadd -rf sailfish-system || :
-usermod -a -G sailfish-system nemo || :
 usermod -a -G sailfish-system sailfish-mdm || :
```
Comment (Tomin1) [inline rpm/sailfish-setup.spec:40]: While at it, could you add this usermod to the list of Requires(pre) commands, please? Others seem to be there already.

Comment (mkenttala) [reply rpm/sailfish-setup.spec:40]: Done

**Pull request merged**